engine/pkg/meta/internal/sqlkv/util.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sqlkv
+
+import (
+	"github.com/pingcap/tiflow/engine/pkg/meta/model"
+	"github.com/pingcap/tiflow/pkg/errors"
+	"github.com/pingcap/tiflow/pkg/errorutil"
+)
+
+// sqlError wraps IsRetryable to etcd error.
+type sqlError struct {
+	displayed error
+	cause     error
+}
+
+func (e *sqlError) IsRetryable() bool {
+	if e.cause != nil {
+		return errorutil.IsRetryableEtcdError(e.cause)
```

Comment:
Should use other func like `IsRetryableSQLError`.

Reply:
Mistaken. I have removed it and add the todo comments.